src/components/CategorySelector.tsx

```diff
@@ -179,7 +215,7 @@ export const CategoryChip = ({ category, onClickChip }: CategoryChipProps) => {
               />
             ) : undefined
           }
-          variant={category.selected ? 'filled' : 'outlined'}
+          variant={category.selected || isHighlightedNviCategory(category.value) ? 'filled' : 'outlined'}
```

Comment:
tror det er dette som lager feilen, sånn konseptuelt.
Dette burde bare baseres på om kategorien er `selected`, og når man klikker på "Kan gi publiseringspoeng" så burde de 4 det gjelder bli satt som selected på samme måte som alle andre kategoriene
```suggestion
          variant={category.selected ? 'filled' : 'outlined'}
```